library/Fission/CLI/Command/Up.hs

```diff
@@ -37,12 +36,41 @@ command cfg =
   addCommand
     "up"
     "Keep your current working directory up"
-    (\dir -> runRIO cfg $ up dir)
-    (strArgument $ mconcat
+    (runRIO cfg . up)
+    commandOptionsParser
+
+
+data CommandOptions = CommandOptions
+  { optDnsOnly :: Bool
+  , optLocation :: String
+  }
```

Comment:
We're keeping types totally separate. Please move to a separate file, maybe `Fission.CLI.Command.Up.Types`

Reply:
Done 👍